universal-application-tool-0.0.1/test/services/applicant/question/AddressQuestionTest.java

```diff
@@ -95,21 +95,22 @@ public void withInvalidApplicantData_missingRequiredFields() {
 
     assertThat(addressQuestion.hasTypeSpecificErrors()).isTrue();
     assertThat(addressQuestion.getStreetErrors())
-        .contains(ValidationErrorMessage.create("Street is required."));
+        .contains(ValidationErrorMessage.create("Please enter valid street name and number."));
```

Comment:
question: Should we use the message constants here too?

Reply:
Good call!